nixos/modules/system/boot/loader/generations-dir/generations-dir-builder.sh

```diff
@@ -62,25 +63,29 @@ addEntry() {
     if test -n "@copyKernels@"; then
         copyToKernelsDir $kernel; kernel=$result
         copyToKernelsDir $initrd; initrd=$result
+        relative="r"
```

Comment:
We should initialise this variable to an empty string to avoid pollution from the environment.